src/app/pages/contact-page/contact-form/contact-form.component.scss

```diff
@@ -10,12 +12,8 @@
   display: block;
   box-sizing: border-box;
   padding: 10px;
-}
-
-.contact-container {
-  box-sizing: border-box;
-  padding: 10px;
-  width: 400px;
+  background-color: #fff;
+  box-shadow: 5px 7px 30px rgb(72 73 121);
```

Comment:
I think at the moment the image in the box is pretty ugly.
We only really have 2 options. Either we go back to the original version without the box.
Or we can maybe try to make the box and image the same size as the form. You can try this to see how it looks. Also, add border-radius: 5px for the round borders to the box.